tigerpath/majors_and_certificates/majors/AAS_2018.json

```
@@ -0,0 +1,815 @@
+                "HUM 310/COM 371/URB 311/ARC 330",
+                "HIS 443/AAS 443",
+                "LAS 217/POR 217/AFS 217/ANT 339",
+                "LAS 313/LAO 313/AAS 331",
+                "LAS 395/AAS 396",
+                "MUS 246/AFS 246",
+                "MUS 258/AFS 258",
+                "MUS 350/AFS 350/ANT 373",
+                "NES 316/AAS 324/HIS 299/JDS 316",
+                "NES 374/GSS 343",
+                "NES 395/HIS 457/AFS 412/AAS 412",
+                "NES 397",
+                "POL 366/AFS 366",
+                "POL 420/AMS 420/AAS 420",
+                "POL 432/AFS 432",
+                "POL 433/AFS 433/GSS 543",
+                "POR 222/LAS 211",
+                "POR 304/LAS 311",
+                "POR 309/LAS 393",
+                "REL 310/AAS 310",
+                "REL 328/GSS 328",
+                "SOC 210/LAS 210/URB 210/LAO 210",
+                "SOC 216",
+                "SOC 227/URB 227",
+                "SOC 327/LAO 328",
+                "SOC 354/AFS 354",
+                "SPA 233/LIN 233/LAS 233",
+                "SPA 327/URB 327/LAO 327",
+                "SPA 352/AAS 352/LAS 356",
+                "SPA 365/LAO 365/URB 365",
+                "SPA 366/AMS 326/LAO 366/GSS 364",
+                "SPA 387/AAS 387",
+                "SWA 200/COM 200"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Independent Work",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "During the fall term all juniors must enroll in AAS 300 Junior Seminar: Research and Writing in African American Studies. This course introduces students to theories and methods of research design in African American Studies in preparation for the junior paper. In the spring term, juniors will complete independent work that includes independent reading and the writing of the junior paper working with a faculty adviser.",
+      "pdfs_allowed": false,
+      "course_list": [
+        "AAS 300"
+      ]
+    },
+    {
+      "name": "Senior Colloquium",
+      "max_counted": null,
+      "min_needed": null,
+      "explanation": "<i>Editor's Note: Concentrators of this major do not enroll in any numbered course for this requirement, so TigerPath does not check it.</i>\n\nConcentrators are required to participate in the <a href=\"https://aas.princeton.edu/academics/undergraduate-program/senior-colloquium\">Senior Colloquium</a>, which seeks to provide a space for seniors concentrating in African American Studies to reflect upon their experiences within the Department, and upon how the understanding and insight they have gained here can and should influence their lives beyond graduation. The Senior Colloquium(link is external) meets a total of six times per term.  A member of the AAS core faculty leads each colloquium meeting, and senior thesis advisors are expected to attend at least one session at the invitation of their advisee.",

```

Just removing the "(link is external)" here
```suggestion
      "explanation": "<i>Editor's Note: Concentrators of this major do not enroll in any numbered course for this requirement, so TigerPath does not check it.</i>\n\nConcentrators are required to participate in the <a href=\"https://aas.princeton.edu/academics/undergraduate-program/senior-colloquium\">Senior Colloquium</a>, which seeks to provide a space for seniors concentrating in African American Studies to reflect upon their experiences within the Department, and upon how the understanding and insight they have gained here can and should influence their lives beyond graduation. The Senior Colloquium meets a total of six times per term.  A member of the AAS core faculty leads each colloquium meeting, and senior thesis advisors are expected to attend at least one session at the invitation of their advisee.",

```